src/index.html

```diff
@@ -0,0 +1,4 @@
+<!DOCTYPE html>
+
+Hello, world!
```

Comment:
Let's add base HTML markup to your code. You can use a shortcut in your editor - type `!` in an empty file and press `tab` button